### PR TITLE
Add JSON RPC network_id

### PR DESCRIPTION
**Pull request by July31Junho**
Add JSON RPC API for querying network_id which client is using #372

### CODE REVIEW

**Comment by sgkim126** [comment]
@July31Junho The spec has been changed on ac22f90bfde9acc71926cb73e214c486a62ff200.

**Comment by July31Junho** [comment]
@sgkim126 I changed chain_getNetworkId RPC Method

**Comment by sgkim126** [comment]
@July31Junho Would you squash the three commits and rebase it on the current master?

**Comment by July31Junho** [comment]
@sgkim126 Of course I'll do this

**Comment by sgkim126** [comment]
Please run rustfmt also. https://github.com/CodeChain-io/codechain/wiki/Coding-Style#formatting-tool

**Comment by July31Junho** [comment]
@sgkim126 I squash the tree commits! This is very interesting!

**Comment by sgkim126** [comment]
Were there four commits? I meant squashing all commits include 8e8f3f372612f05d08b60e2b01e3e4d7c72a91c3 and leaving only one commit.

**Comment by sgkim126** [comment]
And it seems that you don't include the changes of rustfmt.

**Comment by July31Junho** [comment]
@sgkim126 I merged four commit into one commit !

File: rpc/src/v1/traits/net.rs [hunk 1]
```diff
@@ -30,5 +30,8 @@ build_rpc_trait! {
 
         # [rpc(name = "net_isConnected")]
         fn is_connected(&self, ::std::net::IpAddr, u16) -> Result<bool>;
+
```
Comment (sgkim126) [inline rpc/src/v1/traits/net.rs]: Please remove this changes.

**Comment by July31Junho** [comment]
@sgkim126 I removed unnecessarily changed file  (rpc/src/v1/traits/net.rs) !

**Comment by sgkim126** [comment]
Fixes #372

**Pull request merged**